armi/reactor/components/basicShapes.py

```diff
@@ -17,8 +17,14 @@
 
 Many reactor components can be described in 2D by circles, hexagons, rectangles, etc. These
 are defined in this subpackage.
-"""
 
+.. impl:: ARMI shall support a reasonable set of basic shapes.
```

Comment:
This implementation reads like a requirement, which may be kinda confusing. Maybe say "Support a reasonable set of basic shapes"?

Reply:
Ooooh, yeah. I bet I did that in a few places.  Good call!

Reply:
Fixed